llvm/tools/llvm-snippy/README.md

```diff
@@ -0,0 +1,252 @@
+# LLVM-snippy
+
+LLVM-snippy is a cross-platform random code generator. Generation can be model-based or operate in an unmanaged mode. It supports the generation of loops, function calls, memory patterns, and much more.
+
+## Building and Installing the Generator
+
+In this section, we'll cover the steps for building and installing the llvm-snippy generator.
+
+Suppose you have downloaded the source code into the folder ${SNIPPY_PATH}.
+
+Create a file ${SNIPPY_PATH}/release.cmake.
+
+Also, decide on the installation location. I will denote it as /path/to/install for reference, and will refer to it as ${SNIPPY_INSTALL} from now on.
+
+```
+set(CMAKE_BUILD_TYPE RELEASE CACHE STRING "")
+set(CMAKE_INSTALL_PREFIX "/path/to/install" CACHE STRING "")
+set(CMAKE_C_COMPILER clang CACHE STRING "")
+set(CMAKE_CXX_COMPILER clang++ CACHE STRING "")
+set(LLVM_ENABLE_LLD ON CACHE BOOL "")
+set(LLVM_ENABLE_PROJECTS
+  lld
+  clang
+  CACHE STRING "")
```

Comment:
I don't think these projects are necessary for building snippy.

Reply:
Sure. Also removed X86